src/attribute.rs

```diff
@@ -39,7 +39,7 @@ impl AttributeElementExt for Element {
     /// use minidom::Element;
     /// use minidom_ext::AttributeElementExt;
     ///
-    /// let xml: &'static str = r#"<root id="42" />"#;
+    /// let xml: &'static str = r#"<root xmlns="ns" id="42" />"#;
```

Comment:
won't this break some not so valid xml you'd read?

Reply:
It might indeed, but the alternative would be to never update `minidom` (or to get rid of `minidom` entirely which is a much more engaged work).

Reply:
As I was trying to update some other libraries, I discovered that the new version of `minidom` doesn't parse comments anymore...

As a remainder, `minidom` is developed by the XMPP community, and it seems that XML for XMPP have some specificities where comments are forbidden. See this PR for details.
https://gitlab.com/xmpp-rs/xmpp-rs/-/merge_requests/64/diffs

Reply:
seems a bit harsh :/